Modules/Quotas/src/Quotas.Domain/Aggregates/Identities/Identity.cs

```diff
@@ -42,7 +42,7 @@ public IndividualQuota CreateIndividualQuota(MetricKey metricKey, int max, Quota
         return individualQuota;
     }
 
-    public Result<QuotaId, DomainError> DeleteQuota(string individualQuotaId)
+    public Result<QuotaId, DomainError> DeleteIndividualQuota(string individualQuotaId)
```

Comment:
Always use value objects where applicable.
```suggestion
    public Result<QuotaId, DomainError> DeleteIndividualQuota(QuotaId individualQuotaId)
```